src/ngraph_builder.cc

```diff
@@ -1487,6 +1487,17 @@ static Status TranslateFusedBatchNormGradOp(
   SaveNgOp(ng_op_map, op->name(), ng_input_delta_op);
   SaveNgOp(ng_op_map, op->name(), ng_scale_delta_op);
   SaveNgOp(ng_op_map, op->name(), ng_beta_delta_op);
+  // Output reserve_space_3: Unused placeholder to match the mean input
+  // in FusedBatchNorm.
+  std::shared_ptr<ng::Node> output_mean = make_shared<ngraph::op::Constant>(
+      ng_mean->get_element_type(), ng::Shape{}, std::vector<std::string>{""});
```

Comment:
Are you sure scalar is the right shape here?

Reply:
We keep it a scalar for now to match the empty tensor output in tensorflow.